recipes-kernel/linux/linux-fslc/armv8a/defconfig

```diff
@@ -1090,3 +1090,9 @@ CONFIG_DEBUG_KERNEL=y
 # CONFIG_DEBUG_PREEMPT is not set
 # CONFIG_FTRACE is not set
 CONFIG_MEMTEST=y
+CONFIG_KPROBES=y
+CONFIG_OPTPROBES=y
+CONFIG_KRETPROBES=y
+# CONFIG_ARM_KPROBES_TEST is not set
+# CONFIG_KPROBES_SANITY_TEST is not set
```

Comment:
@Jiteshprm  I've already tried and `aarch64` builds are not affected by this, so this change is not needed here.

See https://github.com/Freescale/meta-freescale/pull/668#issuecomment-789796109

Reply:
I'd reply to myself here 😉: This should be kept, as builds for `aarch64` are also affected.

The reason it was not popping up on the first place was the fact that due to `CONFIG_TRACEPOINTS` are not defined by default - `lttng-modules` build was not performed at all. Once `CONFIG_TRACEPOINTS` is enabled - build error occurs also on `aarch64` builds.

@Jiteshprm discard my comment about `aarch64`, you can proceed with the PR.

-- andrey